beacon-chain/attestation/service.go

```diff
@@ -234,35 +232,16 @@ func (a *Service) updateAttestation(ctx context.Context, headRoot [32]byte, beac
 		}
 	}
 
-	cachedCommittees, err = committeeCache.CommitteesInfoBySlot(slot)
+	committee, err := helpers.CrosslinkCommitteeAtEpoch(beaconState, helpers.CurrentEpoch(beaconState), attestation.Data.Shard)
 	if err != nil {
 		return err
 	}
-	if cachedCommittees == nil {
```

Comment:
The committee cache is too tightly coupled with the old spec. Here we temporary remove the committee cache, will add it back once 0.6 is done